server/src/thirdparty/githubAPI/config.js

```diff
@@ -0,0 +1,367 @@
+const service = {
+  name: 'github',
+  triggers: [
```

Comment:
Same here, if config structure is different it will be difficult for framework. If you have issues with slack structure we can discuss and use a common structure.

Reply:
We will be discussing that today